lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/IsAdapter.java

```diff
@@ -65,7 +65,7 @@ private void loadPolicyClassPath(Model model, Helper.loadPolicyLineHandler<Strin
             is.close();
             br.close();
         } catch (IOException e) {
-            e.printStackTrace();
+            LOG.error("IO error occurred");
             throw new Error("IO error occurred");
```

Comment:
While we are here, change this to IllegalStateException and chain the cause

Reply:
fixed in next commit